app/src/main/java/org/dhis2/usescases/settings/models/ErrorModelMapper.kt

```diff
@@ -21,7 +21,7 @@ class ErrorModelMapper(private val fkMessage: String) {
             error.created(),
             error.httpErrorCode().toString(),
             error.errorDescription(),
-            error.errorComponent().name
+            error.errorComponent()?.name ?: ""
```

Comment:
should we show a generic message?

Reply:
Great idea, We should decide what message to show in here.

Reply:
@Balcan Something generic ???

Reply:
@Balcan

Reply:
@JaimeToca @Balcan Talk to @marcamsn about this. For now we will leave the space blank, so we do not need to do anything else in this PR.